tests/tests/test-precompile/test-precompile-staking.ts

```diff
@@ -111,16 +115,19 @@ describeDevMoonbeamAllEthTxTypes("Precompiles - Staking - Collator Leaving", (co
 describeDevMoonbeamAllEthTxTypes("Precompiles - Staking - Join Delegators", (context) => {
   beforeEach("should successfully call delegate for ethan.address to ALITH", async function () {
     await context.createBlock(
-      createTransaction(context, {
-        ...ETHAN_TRANSACTION_TEMPLATE,
-        to: PRECOMPILE_PARACHAIN_STAKING_ADDRESS,
-        data: PARACHAIN_STAKING_INTERFACE.encodeFunctionData("delegate", [
-          alith.address,
-          MIN_GLMR_STAKING,
-          0,
-          0,
-        ]),
-      })
+      createTransaction(
+        context,
+        withoutGasLimit({
```

Comment:
I believe it is easier to do:
```
createTransaction(
        context,
        {
          ...ETHAN_TRANSACTION_TEMPLATE,
          gas: undefined,
          to: PRECOMPILE_PARACHAIN_STAKING_ADDRESS,
          data: PARACHAIN_STAKING_INTERFACE.encodeFunctionData("delegate", [
            alith.address,
            MIN_GLMR_STAKING,
            0,
            0,
          ]),
        })
```

Reply:
Also, removing the gas limit makes it use the estimateGas, that is mostly the difference

Reply:
> removing the gas limit makes it use the estimateGas

+1 yep that's why I removed it